components/operators-detail/documentation/documents-filter.js

```diff
@@ -15,15 +15,23 @@ import {
   getFMUs,
 } from 'selectors/operators-detail/documentation';
 
-function DocumentsFilter({ date, setDate, fmus, FMU, setFMU }) {
+function DocumentsFilter({
+  date,
+  setDate,
+  showDate,
+  fmus,
+  FMU,
+  setFMU,
+  showFMU,
```

Comment:
I think it's a really bad idea to use booleans to show/hide filters. If we think of it as a modular component, it will be better to have an array setting the filters active for each instance. BUT, no time to change it, no worries

Reply:
Agree, this was the quickest way. Easy to refactor, too.